abodepy/helpers/constants.py

```diff
@@ -211,14 +218,11 @@ def get_panel_mode_url(area, mode):
 UNIT_LUX = 'lx'
 LUX = 'lux'
 
+BRIGHTNESS_KEY = 'statusEx'
```

Comment:
This isn't being used anymore since I'm pulling the brightness data from `level` within the `statuses` dictionary of the device JSON data. I figured it's cleaner this way since we're getting the other light information such as hue, saturation and level from there anyways.